lbp.js

```diff
@@ -5,7 +5,7 @@ exports.generateSignature = function(nonce, timestamp, method, uri, apiSecret, b
     // https://docs-blockchain.line.biz/api-guide/Authentication
     var signTarget = `${nonce}${timestamp}${method}${uri}`
 
-    if(body) {
+    if(Object.keys(body).length != 0) {
```

Comment:
@PrayAnalog 
Looks great. but can we use `!_.isEmpty(body)` instead?
since we already imported the module `lodash`

Reply:
Great idea! I applied at a7ff180